### PR TITLE
MNT: Re-rendered with conda-smithy 2.4.0

### PR DESCRIPTION
**Pull request by dopplershift**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,6 @@ env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
-    - CONDA_PY=36
```
Comment (jakirkham) [inline .travis.yml]: Hmm...don't think these should be dropped. At least not Python 3.5 and 3.6.

File: appveyor.yml [hunk 2]
```diff
@@ -17,30 +17,6 @@ environment:
...
-      CONDA_PY: 35
-      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
```
Comment (jakirkham) [inline appveyor.yml]: ...nor these.

**Comment by jakirkham** [comment]
Are we missing some dependencies on Mac and Windows?

File: .travis.yml (hunk 1 above)
Comment (ocefpaf) [inline .travis.yml]: This does not look OK. Let me check that locally.

**Comment by dopplershift** [comment]
I've not seen conda-smithy go sideways like this before.

**Comment by jakirkham** [comment]
Have seen it sometimes, but it usually means there is something missing or in conflict with the build requirements. We need a better way of reporting this information. The trick is this would require a change in behavior for `conda-build-all`.

**Comment by dopplershift** [comment]
Closing in favor of #130.

**Comment by jakirkham** [comment]
Ok, that's fine. Still interested in understanding the cause of this issue.

So I ran `conda smithy rerender` locally and am not seeing these issues. Maybe there is something off with the install. Could you please run `conda export env -n root` and `conda list -n root`?

**Comment by dopplershift** [comment]
@jakirkham I did the rerender again and got different results (that look like #130). But here's what you asked for:

```
>conda env export -n root                                                                    [master]
name: root
channels:
- conda-forge
- defaults
dependencies:
- anaconda-client=1.6.5=py_0
- appnope=0.1.0=py36_0
- asn1crypto=0.22.0=py36_0
- backports.shutil_get_terminal_size=1.0.0=py_2
- backports_abc=0.5=py36_0
- beautifulsoup4=4.6.0=py36_0
- bzip2=1.0.6=1
- ca-certificates=2017.7.27.1=0
- certifi=2017.7.27.1=py36_0
- cffi=1.10.0=py36_0
- chardet=3.0.4=py36_0
- clyent=1.2.2=py36_0
- conda=4.3.23=py36_0
- conda-build=2.1.17=py36_0
- conda-build-all=1.1.1=py_2
- conda-env=2.6.0=0
- conda-smithy=2.4.0=py_1
- conda-verify=2.0.0=py36_0
- cryptography=2.0.3=py36_0
- decorator=4.1.2=py36_0
- entrypoints=0.2.3=py36_1
- feedstockrot=1.0.0=py36_1
- ffmpeg=3.2.4=1
- filelock=2.0.6=py36_0
- flake8=3.4.1=py36_0
- flake8-import-order=0.12=py36_0
- gitdb=0.6.4=py36_1
- gitdb2=2.0.2=py36_0
- gitpython=2.1.5=py36_0
- idna=2.5=py36_0
- ipykernel=4.6.1=py36_0
- ipython=6.1.0=py36_0
- ipython_genutils=0.2.0=py36_0
- jedi=0.10.2=py36_0
- jinja2=2.9.6=py36_0
- jsonschema=2.6.0=py36_0
- jupyter_client=5.1.0=py36_0
- jupyter_core=4.3.0=py36_0
- libconda=4.0.2=py36_0
- libffi=3.2.1=3
- libiconv=1.14=4
- libsodium=1.0.10=0
- markupsafe=1.0=py36_0
- mccabe=0.6.1=py36_0
- mock=2.0.0=py36_0
- nbformat=4.4.0=py36_0
- ncurses=5.9=10
- openssl=1.0.2l=0
- packaging=16.8=py36_0
- path.py=10.3.1=py36_0
- pbr=3.1.1=py36_0
- pep8-naming=0.4.1=py36_0
- pexpect=4.2.1=py36_0
- pickleshare=0.7.4=py36_0
- pip=9.0.1=py36_0
- pkginfo=1.4.1=py36_0
- prompt_toolkit=1.0.15=py36_0
- psutil=5.2.2=py36_0
- ptyprocess=0.5.2=py36_0
- pycodestyle=2.3.1=py36_0
- pycosat=0.6.2=py36_0
- pycparser=2.18=py36_0
- pycrypto=2.6.1=py36_1
- pyflakes=1.5.0=py36_0
- pygithub=1.34=py36_0
- pygments=2.2.0=py36_0
- pyjwt=1.5.2=py36_0
- pyopenssl=17.2.0=py36_0
- pyparsing=2.2.0=py36_0
- pysocks=1.6.7=py36_0
- python=3.6.2=0
- python-dateutil=2.6.1=py36_0
- pytz=2017.2=py36_0
- pyyaml=3.12=py36_1
- pyzmq=16.0.2=py36_2
- readline=6.2=0
- requests=2.18.4=py36_1
- ruamel.yaml=0.15.33=py36_0
- ruamel_yaml=0.11.14=py36_0
- setuptools=36.3.0=py36_0
- simplegeneric=0.8.1=py36_0
- six=1.10.0=py36_1
- smmap=0.9.0=py36_1
- smmap2=2.0.3=py36_0
- sqlite=3.13.0=1
- tk=8.5.19=2
- tornado=4.5.2=py36_0
- tqdm=4.15.0=py36_0
- traitlets=4.3.2=py36_0
- urllib3=1.22=py36_0
- wcwidth=0.1.7=py36_0
- wheel=0.29.0=py36_0
- x264=20131217=3
- xz=5.2.3=0
- yaml=0.1.6=0
- zeromq=4.2.1=1
- zlib=1.2.11=0
- libgcc=4.8.5=1
- pip:
  - backports-abc==0.5
  - backports.shutil-get-terminal-size==1.0.0
  - ipython-genutils==0.2.0
  - jupyter-client==5.1.0
  - jupyter-core==4.3.0
  - prompt-toolkit==1.0.15
  - ruamel-yaml==0.11.14
prefix: /Users/rmay/miniconda3

>conda list -n root                                                                          [master]
# packages in environment at /Users/rmay/miniconda3:
#
anaconda-client           1.6.5                      py_0    conda-forge
appnope                   0.1.0                    py36_0    conda-forge
asn1crypto                0.22.0                   py36_0    conda-forge
backports.shutil_get_terminal_size 1.0.0                      py_2    conda-forge
backports_abc             0.5                      py36_0    conda-forge
beautifulsoup4            4.6.0                    py36_0    conda-forge
bzip2                     1.0.6                         1    conda-forge
ca-certificates           2017.7.27.1                   0    conda-forge
certifi                   2017.7.27.1              py36_0    conda-forge
cffi                      1.10.0                   py36_0    conda-forge
chardet                   3.0.4                    py36_0    conda-forge
clyent                    1.2.2                    py36_0    conda-forge
conda                     4.3.23                   py36_0    conda-forge
conda-build               2.1.17                   py36_0    conda-forge
conda-build-all           1.1.1                      py_2    conda-forge
conda-env                 2.6.0                         0    conda-forge
conda-smithy              2.4.0                      py_1    conda-forge
conda-verify              2.0.0                    py36_0    conda-forge
cryptography              2.0.3                    py36_0    conda-forge
decorator                 4.1.2                    py36_0    conda-forge
entrypoints               0.2.3                    py36_1    conda-forge
feedstockrot              1.0.0                    py36_1    conda-forge
ffmpeg                    3.2.4                         1    conda-forge
filelock                  2.0.6                    py36_0    conda-forge
flake8                    3.4.1                    py36_0    conda-forge
flake8-import-order       0.12                     py36_0    conda-forge
gitdb                     0.6.4                    py36_1    conda-forge
gitdb2                    2.0.2                    py36_0    conda-forge
gitpython                 2.1.5                    py36_0    conda-forge
idna                      2.5                      py36_0    conda-forge
ipykernel                 4.6.1                    py36_0    conda-forge
ipython                   6.1.0                    py36_0    conda-forge
ipython_genutils          0.2.0                    py36_0    conda-forge
jedi                      0.10.2                   py36_0    conda-forge
jinja2                    2.9.6                    py36_0    conda-forge
jsonschema                2.6.0                    py36_0    conda-forge
jupyter_client            5.1.0                    py36_0    conda-forge
jupyter_core              4.3.0                    py36_0    conda-forge
libconda                  4.0.2                    py36_0    conda-forge
libffi                    3.2.1                         3    conda-forge
libgcc                    4.8.5                         1    defaults
libiconv                  1.14                          4    conda-forge
libsodium                 1.0.10                        0    conda-forge
markupsafe                1.0                      py36_0    conda-forge
mccabe                    0.6.1                    py36_0    conda-forge
mock                      2.0.0                    py36_0    conda-forge
nbformat                  4.4.0                    py36_0    conda-forge
ncurses                   5.9                          10    conda-forge
openssl                   1.0.2l                        0    conda-forge
packaging                 16.8                     py36_0    conda-forge
path.py                   10.3.1                   py36_0    conda-forge
pbr                       3.1.1                    py36_0    conda-forge
pep8-naming               0.4.1                    py36_0    conda-forge
pexpect                   4.2.1                    py36_0    conda-forge
pickleshare               0.7.4                    py36_0    conda-forge
pip                       9.0.1                    py36_0    conda-forge
pkginfo                   1.4.1                    py36_0    conda-forge
prompt_toolkit            1.0.15                   py36_0    conda-forge
psutil                    5.2.2                    py36_0    conda-forge
ptyprocess                0.5.2                    py36_0    conda-forge
pycodestyle               2.3.1                    py36_0    conda-forge
pycosat                   0.6.2                    py36_0    conda-forge
pycparser                 2.18                     py36_0    conda-forge
pycrypto                  2.6.1                    py36_1    conda-forge
pyflakes                  1.5.0                    py36_0    conda-forge
pygithub                  1.34                     py36_0    conda-forge
pygments                  2.2.0                    py36_0    conda-forge
pyjwt                     1.5.2                    py36_0    conda-forge
pyopenssl                 17.2.0                   py36_0    conda-forge
pyparsing                 2.2.0                    py36_0    conda-forge
pysocks                   1.6.7                    py36_0    conda-forge
python                    3.6.2                         0    conda-forge
python-dateutil           2.6.1                    py36_0    conda-forge
pytz                      2017.2                   py36_0    conda-forge
pyyaml                    3.12                     py36_1    conda-forge
pyzmq                     16.0.2                   py36_2    conda-forge
readline                  6.2                           0    conda-forge
requests                  2.18.4                   py36_1    conda-forge
ruamel.yaml               0.15.33                  py36_0    conda-forge
ruamel_yaml               0.11.14                  py36_0    conda-forge
setuptools                36.3.0                   py36_0    conda-forge
simplegeneric             0.8.1                    py36_0    conda-forge
six                       1.10.0                   py36_1    conda-forge
smmap                     0.9.0                    py36_1    conda-forge
smmap2                    2.0.3                    py36_0    conda-forge
sqlite                    3.13.0                        1    conda-forge
tk                        8.5.19                        2    conda-forge
tornado                   4.5.2                    py36_0    conda-forge
tqdm                      4.15.0                   py36_0    conda-forge
traitlets                 4.3.2                    py36_0    conda-forge
urllib3                   1.22                     py36_0    conda-forge
wcwidth                   0.1.7                    py36_0    conda-forge
wheel                     0.29.0                   py36_0    conda-forge
x264                      20131217                      3    conda-forge
xz                        5.2.3                         0    conda-forge
yaml                      0.1.6                         0    conda-forge
zeromq                    4.2.1                         1    conda-forge
zlib                      1.2.11                        0    conda-forge
```

**Pull request closed without merging**